niap-cc/NIAPSecExample/app/src/main/java/com/android/certifications/niap/MainActivity.java

```diff
@@ -58,6 +63,13 @@
 import static com.android.certifications.niap.EncryptedDataService.START_FOREGROUND_ACTION;
 import static com.android.certifications.niap.EncryptedDataService.STOP_FOREGROUND_ACTION;
 
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocketFactory;
+import javax.net.ssl.TrustManager;
+
+import info.guardianproject.netcipher.NetCipher;
```

Comment:
Is this actually being used? I didn't see it anywhere else in the diff output.

Reply:
I removed some unused code.

Reply:
Looks good; thanks!